client/src/components/existing-request/existing-request-search.vue

```diff
@@ -1,68 +1,107 @@
 <template>
-  <v-form v-model="isValid" lazy-validation @submit="handleSubmit()" class="pa-10" ref="existing-nr-form">
+  <div>
     <v-row no-gutters>
-      <!-- FIRST LINE -->
-      <v-col cols="12" class="h6 font-weight-regular">
-        Enter the Name Request (NR) Number and either the Applicant's Phone Number or Email:
-      </v-col>
-    </v-row>
 
-    <!-- SECOND LINE -->
-    <v-row class="mt-5" no-gutters v-if="errorMessage">
-      <v-col cols="12" class="red--text" v-html="errorMessage" />
-    </v-row>
-
-    <!-- THIRD LINE -->
-    <v-row class="mt-5" no-gutters align="center">
-      <v-col cols="1" class="max-width">
-        <v-img src="../../assets/images/one-icon.png" contain width="34" height="34" />
-      </v-col>
-      <v-col class="max-height">
-        <v-text-field :rules="nrRules"
-                      :value="search.nrNum"
-                      @input="setExistingRequestSearch('nrNum', $event)"
-                      class="copy-normal"
-                      filled
-                      id="nr-num-text-field"
-                      label="NR Number"
-                      validate-on-blur />
+      <!-- Help Info -->
+      <v-col cols="3" class="existing-request-info copy-small">
+        <div class="ma-8">
+          <ul class="mt-3">
+            <li>
+              Check review time and approval status
+            </li>
+            <li class="test-arrow">
```

Comment:
Is this class used?
(Also, how did you implement the arrow?)

Reply:
sorry, was fiddling here. This needs to be removed. 

I added the arrow to the parent column using `:after`

Reply:
Cool.
How about adding a comment to the CSS block to say what it's doing?

Reply:
Absolutely, will do!